CHANGELOG.md

```diff
@@ -1,4 +1,30 @@
 ## Unreleased
+Upgrade notes:
+* node_files argument added: Due to possible terraform `sensitive` issues, the files that are to be uploaded to a node will need to be specified separately. 
+  For existing code change as follows:
+    ```terraform
+      nodes         = [{
+        node_ip             = "172.31.140.18"
+        node_subnet_id      = data.aws_subnet.subnet1.id
```

Comment:
Won't node IP be there in the older code as well.

Reply:
This has been taken care of.